worth2/templates/main/facilitator_sign_in_participant.html

```diff
@@ -13,36 +14,14 @@
           action="">
         {% csrf_token %}
 
+        {{ form.errors }}
+
         <div class="form-group">
-            <label for="participant_id"
-                   class="control-label col-sm-3">Participant ID #</label>
-            <div class="col-sm-6">
-                <select name="participant_id"
-                        id="participant_id"
-                        class="form-control">
-                    {% for participant in active_participants %}
-                    <option value="{{participant.pk}}">
-                        {{participant.study_id}}
-                    </option>
-                    {% endfor %}
-                </select>
-            </div>
+            {{ form.participant_id|bootstrap_horizontal:'col-sm-3' }}
```

Comment:
Oh right - I forgot we talked about that! I should have PMT'd myself. Now we're talking about allowing the facilitator to filter down this participant dropdown by selecting from a Cohort dropdown box.